### PR TITLE
docs: add 7.1-7.15 weekly report

### PR DESCRIPTION
**Pull request by lijing-21**
add 7.1-7.15 weekly report

### CODE REVIEW

File: website/static/data/events.json [hunk 1]
```diff
@@ -1,5 +1,9 @@
 [   
   {
+    "title": "Biweekly Report (Jul 1 - Jul 30)",
```
Comment (yzeng25) [inline website/static/data/events.json]: I think you mean July 15th here.
```suggestion
    "title": "Biweekly Report (Jul 1 - Jul 15)",
```

File: blog/zh/blog/2022/07/21/weekly-report-0715.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+https://pkg.go.dev/k8s.io/apimachinery/pkg/util/wait 包提供了轮询（Polling）或监听条件变化的工具。
+
+可以用 `wait` 代替当前的 `time.sleep`，以一种更好的方式来确定是否已经达到我们所期望的状态。
+
+## 近期功能特性亮点
+
+- [新增在 Ingress 资源中可以通过 annotation 绑定 ApisixPluginConfig 的能力，让 Ingress 资源可以直接使用 APISIX 的插件能力](https://github.com/apache/apisix-ingress-controller/pull/1139)（贡献者：[dickens7](https://github.com/dickens7)）
+
+- [- 修复 APISIX Dashboard 中路由复制时的 ID 冲突问题](https://github.com/apache/apisix-dashboard/pull/2501)（贡献者：[SkyeYoung](https://github.com/SkyeYoung)）
```
Comment (yzeng25) [inline blog/zh/blog/2022/07/21/weekly-report-0715.md]: Remove extra bullet point mark.

image


```suggestion
- [修复 APISIX Dashboard 中路由复制时的 ID 冲突问题](https://github.com/apache/apisix-dashboard/pull/2501)（贡献者：[SkyeYoung](https://github.com/SkyeYoung)）
```

File: website/static/data/events.json (hunk 1 above)
Comment (yzeng25) [reply website/static/data/events.json]: image

File: blog/zh/blog/2022/07/21/weekly-report-0715.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+“独行者速，众行者远”。Apache APISIX 社区周报希望可以帮助社区小伙伴们更好地掌握 Apache APISIX 社区的进展，方便大家参与到 Apache APISIX 社区中来。
+
+我们还整理了一些适合新来社区的小伙伴们参加的 issue！感兴趣的同学们，走过路过不要错过！
+
+## 贡献者统计
+
+![本周贡献者名单](https://static.apiseven.com/2022/weeklyreport/0721/3.jpg)
+
+![本周新晋贡献者](https://static.apiseven.com/2022/weeklyreport/0721/1.png)
```
Comment (yzeng25) [inline blog/zh/blog/2022/07/21/weekly-report-0715.md]: Borbur already started contributed on or before June 30th, see https://apisix.apache.org/blog/2022/06/21/weekly-report-0621/#contributor-statistics.

Why is he listed as 'New contributor' here?

File: blog/en/blog/2022/07/21/weekly-report-0715.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+"If you want to go fast, go alone.If you want to go far, go together." The Apache APISIX Community Weekly Report hopes to help community members better understand the weekly progress of the Apache APISIX community and and facilitate everyone to participate in the Apache APISIX community.
+
+We have also compiled some issues suitable for newcomers to the community to participate in! If you are interested, don't miss it!
+
+## Contributor Statistics
+
+![Contributors List](https://static.apiseven.com/2022/weeklyreport/0721/4.jpg)
+
+![New Contributors](https://static.apiseven.com/2022/weeklyreport/0721/2.png)
```
Comment (yzeng25) [inline blog/en/blog/2022/07/21/weekly-report-0715.md]: Borbur already started contributed on or before June 30th, see https://apisix.apache.org/blog/2022/06/21/weekly-report-0621/#contributor-statistics.

Why is he listed as 'New contributor' here?

File: blog/en/blog/2022/07/21/weekly-report-0715.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+### Issue #1146
+
+**Link**: https://github.com/apache/apisix-ingress-controller/issues/1146
+
+**Description**: We should add a new `make dev-env entry` in Makefile, to start a dev environment.
+
+It should do something:
+
+- using kind create a new cluster
```
Comment (SylviaBABY) [inline blog/en/blog/2022/07/21/weekly-report-0715.md]:
```suggestion
- Use kind create a new cluster
```

File: blog/en/blog/2022/07/21/weekly-report-0715.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+- Deploy APISIX + etcd
+- Build `apisix-ingress-controller`’s image
+- Deploy `apisix-ingress-controller`
+
+Most of them are already contained in the current Makefile, we need to organize them
```
Comment (SylviaBABY) [inline blog/en/blog/2022/07/21/weekly-report-0715.md]:
```suggestion
Most of them are already contained in the current Makefile, we need to organize them.
```

File: blog/zh/blog/2022/07/21/weekly-report-0715.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
+---
+title: "社区双周报 7.1 - 7.15"
```
Comment (SylviaBABY) [inline blog/zh/blog/2022/07/21/weekly-report-0715.md]:
```suggestion
title: "社区双周报（7.1 - 7.15）"
```

File: blog/zh/blog/2022/07/21/weekly-report-0715.md [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+**问题描述**：应该在 Makefile 中添加一个新的 `make dev env`，以启动开发环境。
+
+它需要完成：
+
+- 使用 kind 创建新集群
+- 部署 APISIX + etcd
+- 构建 `apisix-ingress-controller` 的镜像
+- 部署 `apisix-ingress-controller`
+
+其中大多数已经包含在当前的 Makefile 中，需要对其进行组织
```
Comment (SylviaBABY) [inline blog/zh/blog/2022/07/21/weekly-report-0715.md]:
```suggestion
其中大多数已经包含在当前的 Makefile 中，需要对其进行组织。
```

File: blog/en/blog/2022/07/21/weekly-report-0715.md (hunk 4 above)
Comment (lijing-21) [reply blog/en/blog/2022/07/21/weekly-report-0715.md]: sorry, forgot to update.

**Review by SylviaBABY — APPROVED** [review]

**Review by yzeng25 — APPROVED** [review]

**Pull request merged**